content/100-getting-started/01-quickstart.mdx

```diff
@@ -13,7 +13,7 @@ The following guide will get you up and running as quickly as possible by coveri
 2. Install your dependencies.
 3. Query some data and view the results.
 
-> **Note**: You need [**Node.js**](https://nodejs.org/en/) (version 12.6 or higher) for this tutorial.
+> **Note**: You need [**Node.js**](https://nodejs.org/en/) (version 14.0 or higher) for this tutorial.
```

Comment:
Might be worth adding a link to the sys requirements page here.

Reply:
Good point, will do